packages/sdk/src/response-time-processor.ts

```diff
@@ -30,7 +31,7 @@ export class SwResponseTimeProcessor extends NoopSpanProcessor {
   }
 
   override onEnd(span: ReadableSpan): void {
-    if (!isEntrySpan(span)) return
+    if (!isEntrySpan(span) || span.kind !== SpanKind.SERVER) return
```

Comment:
Our existing inbound metrics logic is actually a bit different--we do collect the response time for any "entry span" but only add the HTTP method and status attributes if it's a SERVER span kind.

I agree it doesn't seem useful to report the metric in case of a "detached" CLIENT span like an AWS SDK call, but it is useful for CONSUMER entry spans. Let's keep with the existing inbound metrics logic for now, we might fine-tune the response time logic as a whole later.

Reply:
Addressed in #288